contract/package.json

```diff
@@ -56,6 +57,7 @@
     "@agoric/vats": "0.15.2-u14.0",
     "@agoric/zoe": "^0.26.3-u14.0",
     "@endo/bundle-source": "^2.8.0",
+    "@endo/exo": "^1.3.0",
```

Comment:
This dapp uses packages from [agoric-upgrade-14](https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-14). The version there seems to be "@endo/exo@0.2.2".

https://github.com/Agoric/agoric-sdk/blob/d69c011001d300735546dcb74a9ae21e306e4789/yarn.lock#L1395

I don't know what happens if you mix in this version of exo.

Reply:
I think I need to wait until I can depend on a more recent endo before proceeding farther. I need at least support for `M.raw()` in order to sidestep some of the exceptional cases.

I am surprised to see lots of async generator definitions in this repo. Looks like twelve! This is the cause of some of the special cases I need to sidestep. Where did these come from?

Reply:
In case `M.raw()` is blocking progress here, it became available in Agoric SDK on March 20. Please let me know if there’s further Endo work in flight to unblock this draft.

Reply:
> This dapp uses packages from [agoric-upgrade-14](https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-14). The version there seems to be "@endo/exo@0.2.2".

> In case M.raw() is blocking progress here, 

It is.

> it became available in Agoric SDK on March 20. Please let me know if there’s further Endo work in flight to unblock this draft.

What is needed so `M.raw()` is available here?